spec/controllers/fines_controller_spec.rb

```diff
@@ -43,7 +43,7 @@
 
     let(:fines) do
```

Comment:
This `fines` variable is identical to the variable on line 116, can we dry it up?